sidebars.js

```diff
@@ -1023,6 +1023,11 @@ const sidebars = {
           id: "articles/slack-integration",
         },
         { type: "doc", label: "Prometheus", id: "articles/metrics" },
+        {
```

Comment:
this looks more operational to me - shouldn't this be under the "Installation/On-Prem Control Plane/Advanced Guides" section?